src/containers/home/index.tsx

```diff
@@ -104,22 +115,54 @@ const Home: React.FC<HomeContainerProps> = ({ events, announcements }) => {
         />
       </Helmet>
 
-      <LandingContainer>
-        <LandingCard bordered={false}>
-          <LandingTextContainer>
-            <LandingText>
-              Welcome to Lucy's Love Bus Event Registration!
-            </LandingText>
-          </LandingTextContainer>
-          <LandingBodyText>
-            The Sajni Center invites you to use this page as your portal to view
-            and register for events, and stay up to date with our community!
-          </LandingBodyText>
-          <LinkButton type="primary" to="/upcoming-events">
-            View Events
-          </LinkButton>
-        </LandingCard>
-      </LandingContainer>
+      {privilegeLevel === PrivilegeLevel.ADMIN ? (
+        <LandingContainer>
+          <LandingCard bordered={false}>
+            <LandingTextContainer>
+              <LandingText>Welcome Administrator!</LandingText>
+            </LandingTextContainer>
+            <LandingBodyText>
+              Explore and use these tools to help create a space of hope and
+              healing for the members of The Sajni Center.
+            </LandingBodyText>
+            <Row>
+              <AdminCol span={8}>
```

Comment:
🔧 In the screenshot it looks like the buttons arent aligned against the left side of the container if that makes sense? Check the Figma but I would rather have the group of buttons left aligned. You can do this with a display: flex instead of using AntD grid components

Reply:
Figma told me to center align 😳

Reply:
But I can left align them sure